.github/workflows/deploy_publish.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Publish
+on:
+  push:
+    branches: [master]
+    paths-ignore:
+      - "**.md"
+
+jobs:
+  docker-build-publish:
+    name: Docker Build & Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Verify app version
+        run: |
+          APP_VERSION=$(cat package/.appversion)
+          if [ $(git tag -l "$APP_VERSION") ]; then 
```

Comment:
defualt depth for checkout@v2 is 1 - that would mean you wont have tags available on the container to query. Use 

```
uses: actions/checkout@v2
        with:
          fetch-depth: 0
```